### DISCUSSION TITLE
Customisations for production using helm config (Environment vars and storage location)

### DISCUSSION THREAD

**Discussion by markmcgookin** [datalust/seq-tickets#1791] [Help]
Hi, 

We have a fairly basic Seq setup that we have installed with the helm chart essentially with no real customisations. We are upgrading our licence to Enterprise so that we can put this in our Test and Production environments.

We have a few things that we would like to customise, which we are familiar with doing with 'normal' kubernetes installs, but not with Helm.

1) Setting the SEQ_API_CANONICALURI Environment variable so that we can setup Slack alerts and have them link into Seq. Is this possible with the Conflig.yaml file [here](https://docs.datalust.co/docs/using-helm) or is it done automatically when specifying the ingress?

2) Setting a custom location for logs. We don't want to just store the logs locally, ideally we would persist them to a storage account (We are using Azure AKS, so I believe we can mount a volume from an Azure storage account as a Volume on the pod) is this possible by this file as well? 

Many thanks

**Comment by KodrAus** [comment]
Hi @markmcgookin :wave:

It looks like we don't currently expose a way to configure the canonical URI through the Helm chart. I've created https://github.com/datalust/helm.datalust.co/issues/34 to track adding support for that.

For storage, you should be able to use [an Azure Disk as a persistent volume](https://learn.microsoft.com/en-us/azure/aks/concepts-storage#persistent-volumes), ideally using the `managed-csi-premium` storage class.

**Reply by KodrAus** [reply]
For reference, the source for the chart [lives here](https://github.com/datalust/helm.datalust.co/tree/main/charts/seq), which you can also dig through to see how the deployment is built from the configuration 🙂

**Reply by markmcgookin** [reply]
That’s great @KodrAus thanks for the help. Got most of it sorted manually with the chart source

**Reply by KodrAus** [reply]
Glad you got most of it sorted! I'd also totally forgotten about this, but we do actually also expose the canonical URI as well already. It's passed through [as the `baseURI` property in your `Values.yaml`](https://github.com/datalust/helm.datalust.co/blob/main/charts/seq/values.yaml#L23).
